catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -8,26 +8,42 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
 
 We will keep this page up to date with the latest information as we make more progress.
 
-## What’s new? (updated 8 July 2020)
+## What’s new? (updated 14 October 2020)
 
-### Improvements to search relevance
+### Full catalogue search
+We were previously only exposing a subset of our catalogue in search results, limited to digital and digitised materials.
+We now return results from our whole catalogue, including archives, physical works from our collections, and more.
 
-#### Fine tuning relevance
-Based on the data we already have about how search is being used, we've tested a change to the way queries are split and combined. Rather than letting results with _any_ (one or more) of the terms appear in your results (using the OR operator between query tokens), we now require that _all_ of the tokens appear in any result you see (using the AND operator between query tokens). This seems to be returning more relevant results, so we've made this the default algorithm, while kicking off a new test.
+This means you will see many more results for a given query, as we will be searching among approximately 1.2 million works, up from Approximately 350,000.
+
+To help you narrow the results our current filters should help and we have added a new search filter.
+
+### Online / In library search filter
+We've added a filter to the search form, which will allow you to choose between online and in library works.
+As there are potentially many more results being returned, this will allow you to narrow those results in another useful way.
```

Comment:
Perhaps "narrow to those results that you can view without having to visit the library".